app/assets/stylesheets/overrides.scss

```diff
@@ -98,9 +98,17 @@ height: 49px;
   background-color: $subnav;
   border-bottom-left-radius: 30%;
   border-bottom-right-radius: 30%;
-  font-size: 10pt;
-  height: 45px;
-  padding-top: 10px;
+  padding: 10px 20px;
+  height: 40px;
+}
+
+.five-up.nav-bottom.show-for-small a {
+  padding-top: 0;
+  height: 50px;
+  font-size: 30px;
+  :hover {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3